pkg/occlient/occlient.go

```diff
@@ -34,18 +35,23 @@ import (
 	"github.com/openshift/source-to-image/pkg/tar"
 	s2ifs "github.com/openshift/source-to-image/pkg/util/fs"
 
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
```

Comment:
This should be named properly. `v1beta1` is too generic and it will collide with other objects.
maybe `catalogv1beta1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"` ?

Reply:
changed to scv1beta1